fast64_internal/oot/__init__.py

```diff
@@ -58,6 +58,13 @@
     oot_operator_unregister,
 )
 
+
+feature_set_enum = (
+    ("decomp", "Decomp", "Decomp"),
+    ("hacker_oot", "HackerOOT", "Hacker OOT"),
```

Comment:
the correct spelling is exactly "HackerOoT" https://github.com/HackerN64/HackerOoT

Reply:
Done.